pkg/webhook/inject/inject.go

```diff
@@ -52,7 +50,7 @@ const (
 func Inject(res *v1.AdmissionRequest, cli client.Client, cfg *config.Config, controllerCfg *controllerCfg.ChaosControllerConfig, metrics *metrics.ChaosControllerManagerMetricsCollector) *v1.AdmissionResponse {
 	var pod corev1.Pod
 	if err := json.Unmarshal(res.Object.Raw, &pod); err != nil {
-		log.Error(err, "Could not unmarshal raw object")
+		log.L().WithName("inject").Error(err, "Could not unmarshal raw object")
```

Comment:
how about defining a variable for `log.L().WithName("inject")`?

Reply:
Updated! PTAl